src/drivers/io/vesa_log.zig

```diff
@@ -55,8 +61,27 @@ const Framebuffer = struct {
   }
 };
 
+fn default_updater(x: u32, y: u32, w: u32, h: u32, ctx: u64) void {} // Do nothing
+
 var framebuffer: ?Framebuffer = null;
 
+pub const Updater = fn(x: u32, y: u32, width: u32, height: u32, ctx: u64)void;
+pub const FBInfo = struct { phys: u64, width: u32, height: u32 }; 
+
+pub fn get_info() ?FBInfo {
+  if (framebuffer) |fb| {
+    var i = .{ .phys = fb.phys, .width = @truncate(u32, fb.width), .height = @truncate(u32, fb.height) };
```

Comment:
Do you still need the `@truncate()` after changing the types in the framebuffer above? I'm fine either with having the `@truncate()` and leaving the types alone, or changing the type and not having `@truncate()`, but having both is strange.

Reply:
On another note, I think `@intCast()` is preferable here.